cocos/dragon-bones/ArmatureCache.ts

```diff
@@ -185,32 +220,54 @@ export class AnimationCache {
 
         this.end();
     }
-
+    /**
+     * @en Check if initialized or not.
+     * @zh 检查是否已初始化。
+     * @returns {Boolean}
+     */
     isInited () {
         return this._inited;
     }
-
+    /**
+     * @en Check if current state is invalid.
+     * @zh 检查当前状态是否为无效。
+     * @returns {Boolean}
```

Comment:
Should add comments about the meaning of the bool. For example, true means xxx, else false.